js/main.js

```diff
@@ -153,3 +177,16 @@ document.addEventListener('DOMContentLoaded', function (event) {
     viewSwapDataViews(data.view);
   }
 });
+
+// eventlistener 'click' for document
+document.addEventListener('click', function (event) {
+
+  if (event.target.tagName !== 'A') {
+    return;
+  }
+
+  if (event.target.tagName === 'A' && data.profile.username) {
```

Comment:
You can remove the first part of this condition. If you reached this line in the code, it is an anchr tag, because if it wasn't, you would have returned from the function.